examples/chat2-reliable/chat.go

```diff
@@ -259,28 +275,48 @@ func (c *Chat) parseInput() {
 }
 
 func (c *Chat) SendMessage(line string) {
-	tCtx, cancel := context.WithTimeout(c.ctx, 3*time.Second)
-	defer func() {
-		cancel()
-	}()
+	c.incLamportTimestamp()
```

Comment:
Shouldn't the part of the send logic that updates reliability constructs be in the reliability.go layer rather than in the app?